pkg/models/clustersConfig.go

```diff
@@ -28,6 +28,7 @@ type ClusterConfig struct {
 	ExtraNetworkData   ExtraNetworkData
 	Subnets            []string
 	External           bool
+	HTTPAccess         constants.Visibility
```

Comment:
pls change Visibility type to something giving more idea of the context, RemoteHostAccess, or smb like that

Reply:
changed to PublicHTTPAccess